bigtable-client-core-parent/bigtable-client-core/src/test/java/com/google/cloud/bigtable/config/TestUserAgent.java

```diff
@@ -127,12 +141,19 @@ public void testDataClientWithSSLContext() throws Exception{
   }
 
   /**
-   * Ignored this test case as it is not being intercepted.
+   * Ignoring this test cases as either of below scenario traps in infinite loop.
```

Comment:
If these are two test cases, reword them as "Ignoring these test cases as either of the below scenarios, traps in an infinite loop." 
If it is a single test case, reword as: "Ignoring this test case as either of the below scenarios, traps in an infinite loop."